launcher/CMakeLists.txt

```diff
@@ -941,7 +941,6 @@ target_link_libraries(Launcher_logic
     Launcher_iconfix
     QuaZip::QuaZip
     hoedown
-    PolyMC_rainbow
```

Comment:
uhhh

Reply:
What the fuck is even going on, I removed that

Reply:
It should be `Launcher_rainbow`. Your second to last force push removed it for some reason